### PR TITLE
Added Rhodox Glass Panes

### PR DESCRIPTION
**Pull request by Damonj17**
This should be all the textures for the sides of the glass panes. Having these set up separately from the glass block textures would be better, in my opinion.

### CODE REVIEW

**Comment by t3rminus** [comment]
So two things: 
The `"name"` field must be unique. It's using this as a key for the selector, so duplicates are ignored (I manually corrected some in the last PR, but I notice there are a few in this one as well. Some are a bit off-- "Chestnut Wood Pane" for Chinese-style window)

Secondly, you don't need `"preview"` if there's only one output file. It will use that by default.
Same with the output... if there's only one file, it doesn't need to be in an array (so `["a1-glasspane-chestnutrpg.png"]` can become just `"a1-glasspane-chestnutrpg.png"`)

**Comment by Piogre** [comment]
I'm curious as to why we want these separate from the main glass selection -- the game forces glass blocks and glass panes to have the same face textures, splitting these off would just make it so someone can have the edge of their glass panes not "match" the face.  Theoretically possible, but very niche, and meanwhile we pass up the ability for someone picking the face of the glass to bundle in the matching edge automatically.

IMO if it means 99% of people don't need to search for the "matching" edge image (said edges being particularly difficult to preview being only 4 pixels wide), that's worth it even if it means the remaining 1% have to do some file pushing to get the intentionally-mismatched result

**Comment by Damonj17** [comment]
Yeah, there were a few names that were duplicates, those weren't on purpose. The names are based on the description, which can sometimes get a bit wordy.

I wanted these separate because of the niche ability. Perhaps it would be better to make it so that it automatically matches the pane textures with the glass, but then include the option for an override?

**Comment by aerottd** [comment]
> IMO if it means 99% of people don't need to search for the "matching" edge image (said edges being particularly difficult to preview being only 4 pixels wide), that's worth it even if it means the remaining 1% have to do some file pushing to get the intentionally-mismatched result

Question: would the edge image not be bundled with the pane / glass block texture automatically, or am I misinterpreting something? I do agree, though, that we should probably keep the pane and glass block textures the same if not only to make the customizer easier to use; at some point people are going to have to just run it twice to get total customization.

**Comment by t3rminus** [comment]
Yeah, I think matching them is preferred. I can't imagine anyone wanting different glass pane edges, and those that do can probably customize it on their own. 
Let's follow the original customizer in this regard and add the page edge textures to the normal glass options.

**Comment by Piogre** [comment]
>I do agree, though, that we should probably keep the pane and glass block textures the same if not only to make the customizer easier to use; at some point people are going to have to just run it twice to get total customization.

Bit of confusion here.

We don't need to do anything to make glass block texture the same as the front face of the glass pane texture.  The game does that automatically; both the glass pane front face and the block face draw from textures/block/glass.png -- we can't make them different unless we mess with the models

>Question: would the edge image not be bundled with the pane / glass block texture automatically, or am I misinterpreting something?

Not currently.  Currently the glass pane uses textures/block/glass.png for the front face and textures/block/glass_pane_top.png for the edges.  This submission would make it so once someone selects the texture to be used for glass blocks (which is also used for the glass pane front face) they'd have to go searching through the list of 4-pixel-wide edge images to find the one that matches.  I think it's better that we skip this, and make it so selecting glass texture automagically bundles in the glass pane texture, and if someone REALLY wants mismatched glass panes they can run it twice.

**Comment by t3rminus** [comment]
OK. I'm going to close this. @Damonj17 If you can re-do your pull request so it adds the pane edges as outputs on the original glass options, and submit a separate pull request for that, and wool, using different branches.

**Comment by aerottd** [comment]
> Not currently. Currently the glass pane uses textures/block/glass.png for the front face and textures/block/glass_pane_top.png for the edges. This submission would make it so once someone selects the texture to be used for glass blocks (which is also used for the glass pane front face) they'd have to go searching through the list of 4-pixel-wide edge images to find the one that matches. I think it's better that we skip this, and make it so selecting glass texture automagically bundles in the glass pane texture, and if someone REALLY wants mismatched glass panes they can run it twice.

Upon reading this I checked the actual game files, and I have to say I agree, not that it matters much now that this is closed. I was thinking 'edge' was the front face (i.e. a different colored edge with the same glass center), but now that I see it's the two-pixel slice, it really does seem overkill to have a specific customization option for it.
